adding_data_long.qmd

```diff
@@ -1266,7 +1267,7 @@ f <- file.path("data", dataset_ids, "metadata.yml")
 read_metadata(f) %>% write_metadata(f)
 
 # rebuild
-remake::make(dataset_ids)
+build(dataset_ids)
```

Comment:
as above

Reply:
Forgot to go back to this after we talked. I can replace it with `source(build.R)`, correct?

Reply:
Yes, though at will build the whole thing